### PR TITLE
fix build warnings

### PR DESCRIPTION
**Pull request by remicollet**
-Wunused-variable and -Wimplicit-function-declaration

### CODE REVIEW

File: src/class.c [hunk 1]
```diff
@@ -24,6 +24,8 @@
 
 #include "util.h"
 #include "class.h"
+#include <Zend/zend_exceptions.h>
```
Comment (krakjoe) [inline src/class.c]: This should be included in uopz.h now.

File: src/constant.c [hunk 2]
```diff
@@ -24,6 +24,7 @@
 
 #include "util.h"
 #include "constant.h"
+#include <Zend/zend_exceptions.h>
```
Comment (krakjoe) [inline src/constant.c]: So should this be in uopz.h

File: src/function.c [hunk 3]
```diff
@@ -27,6 +27,7 @@
 #include "copy.h"
 
 #include <Zend/zend_closures.h>
+#include <Zend/zend_exceptions.h>
```
Comment (krakjoe) [inline src/function.c]: This and the line above are in uopz.h

**Comment by krakjoe** [comment]
Thanks.

I think you need to pull in latest changes, looks out of date ?

**Comment by remicollet** [comment]
yah...

**Pull request closed without merging**